src/components/BusinessView.tsx

```diff
@@ -0,0 +1,104 @@
+import React, {useState} from "react"
+import { SafeAreaView, View, FlatList, Text, Image, StyleSheet, TouchableOpacity, ScrollView } from "react-native"
+import Business from './Business'
+
+const ExampleBusiness: Business = {
```

Comment:
You'll be able to either create a more refined definition using the `Pick` or `Omit` [utility types](https://www.typescriptlang.org/docs/handbook/utility-types.html) and the imported Business type from `@typings/types`.